lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -258,9 +270,19 @@ def main():
                 if meraki.status == 201:
                     meraki.result['data'] = r
                     meraki.result['changed'] = True
-            else:
+            else:  # Update existing network
                 net = meraki.get_net(meraki.params['org_name'], meraki.params['net_name'], data=nets)
-                if meraki.is_update_required(net, payload):
+                if meraki.params['enable_vlans'] is not None:
+                    payload = {'enabled': meraki.params['enable_vlans']}
+                    path = meraki.construct_path('enable_vlans',
+                                                 net_id=meraki.get_net_id(net_name=meraki.params['net_name'], data=nets))
+                    r = meraki.request(path,
+                                       method='PUT',
+                                       payload=json.dumps(payload))
+                    if meraki.status == 200:
+                        meraki.result['data'] = r
+                        meraki.result['changed'] = True
+                elif meraki.is_update_required(net, payload):
```

Comment:
Should you not be able to enable_vlans support, and update the network ?
So I was expecting an `if` instead of an `elif`.

Reply:
I added an example and there already should be integration tests.